README.md

```diff
@@ -37,6 +39,40 @@ You can use this [shell script][bootstrap-linux] to turn your laptop into an awe
 
 TODO
 
+## Run in Docker
+
+If you don't like to bootstrap your machine you can also run a [Docker][docker] container for development by invoking a single command only. The container will be pulled from our registry. You need to have [Docker Engine][docker-engine] installed first.
+
+~~~ bash
+# [pull and] run container with default rake task (eq. build `_site` and serve via jekyll)
+$ docker run --rm --volume=$(pwd):/usr/src/epages-docs -p 127.0.0.1:4000:4000 -it docker.epages.com/epages/docs-ruby
```

Comment:
You can just write `-p 4000:4000`. Then the command is on all systems the same (although on systems using docker-machine the URL one has to enter into the browser is still different from `localhost:4000`).

To then be able to reach the docs, you have to change the interface jekyll binds to in the `Rakefile`:

```ruby
task :serve do
  sh "bundle exec jekyll serve --host 0.0.0.0 --skip-initial-build --watch"
end
```

The downside of this solution, is that the dev server when not running in docker is reachable from the outside. But I would still prefer this solution, as it unifies command line stuff across platforms.